operations/patch_util.go

```diff
@@ -374,26 +388,26 @@ func loadGitData(branch string, ref string, format bool, extraArgs ...string) (*
 	if len(extraArgs) > 0 {
 		statArgs = append(statArgs, extraArgs...)
 	}
-	stat, err := gitDiff(mergeBase, ref, statArgs...)
+	stat, err := gitDiff(mergeBase, ref, rev, statArgs...)
```

Comment:
According to the [git docs](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-emgitdiffemltoptionsgtltcommitgtltcommitgt--ltpathgt82308203) `..` and `...` have a different meaning for `git diff` and [git rev-list](https://git-scm.com/docs/git-rev-list#_description) then they do for `git log` and `git format-patch`.

Reply:
This does seem to be true for branches and not for commit hashes.

Reply:
In my experiments it looked like it's an issue for hashes as well.
Let's talk this over in person.